framework-plugins/lisk-framework-faucet-plugin/src/plugin/faucet_plugin.ts

```diff
@@ -120,14 +136,95 @@ export class FaucetPlugin extends BasePlugin {
 					throw new Error('Password given is not valid.');
 				}
 			},
+			fundTokens: async (params?: Record<string, unknown>): Promise<{ result: string }> => {
+				const errors = validator.validate(fundParamsSchema, params as Record<string, unknown>);
+
+				if (errors.length) {
+					throw new LiskValidationError([...errors]);
+				}
+
+				if (!this._state.publicKey || !this._state.passphrase) {
+					throw new Error('Faucet is not enabled.');
+				}
+
+				const { address } = params as Record<string, unknown>;
+				const encodedTransaction = await this._createTransferTransaction(address as string);
+				await this._channel.invoke<{
+					transactionId?: string;
+				}>('app:postTransaction', {
+					transaction: encodedTransaction,
+				});
+
+				return {
+					result: `Successfully funded account at address: ${address as string}.`,
+				};
+			},
 		};
 	}
 
-	// eslint-disable-next-line @typescript-eslint/require-await, class-methods-use-this, @typescript-eslint/no-empty-function
-	public async load(_channel: BaseChannel): Promise<void> {
-		this._options = objects.mergeDeep({}, defaults.config, this._options) as FaucetPluginOptions;
+	// eslint-disable-next-line @typescript-eslint/require-await, class-methods-use-this
+	public async load(channel: BaseChannel): Promise<void> {
+		this._channel = channel;
+		this._options = objects.mergeDeep(
+			{},
+			defaults.config.default,
+			this._options,
+		) as FaucetPluginOptions;
 	}
 
 	// eslint-disable-next-line class-methods-use-this, @typescript-eslint/no-empty-function
 	public async unload(): Promise<void> {}
+
+	private async _createTransferTransaction(address: string): Promise<string> {
+		// ModuleID: 2 (Token), AssetID:0 (TransferAsset)
+		const transferAssetInfo = this.schemas.transactionsAssets.find(
```

Comment:
you can create api client using channel, and create and send the transaction using the client